lib/src/presentations/on_boarding/screen/on_boarding_page.dart

```diff
@@ -0,0 +1,81 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+
+import '../../../core/constants/constants.dart';
+
+class OnBoardingPage extends StatelessWidget {
+  const OnBoardingPage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final width = MediaQuery.sizeOf(context).width;
+
+    return Scaffold(
+      floatingActionButton: FloatingActionButton(
+        onPressed: () {},
+        shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(360)),
+        elevation: 0,
+        child: const Icon(FontAwesomeIcons.arrowRight),
+      ),
+      body: SafeArea(
```

Comment:
Should be separate `SafeArea()` with function `_body(){ return SafeArea()}`

Reply:
I fixed it